packages/web-app-preview/src/components/Sources/MediaImage.vue

```diff
@@ -26,6 +29,63 @@ export default defineComponent({
       type: Number,
       required: true
     }
+  },
+  setup(props) {
+    const img = ref()
```

Comment:
Could you update this again from current master? @JammingBen and myself changed this today.

Reply:
Sure